orm_lib/src/sqlite3_impl/Sqlite3Connection.cc

```diff
@@ -172,7 +172,7 @@ void Sqlite3Connection::execSqlInQueue(
     auto stmt = stmtPtr.get();
     for (int i = 0; i < (int)parameters.size(); ++i)
     {
-        int bindRet;
+        int bindRet = SQLITE_ERROR;
```

Comment:
int bindRet{SQLITE_OK};

Reply:
Done. But wouldn't that cause bugs if `format[i]` is invalid? (Well, `format[i]` being invalid is a bug of itself, but it'll be safer if we assume it can be wrong)